src/assets/translations/oc.json

```diff
@@ -20,7 +20,7 @@
         "cantLoadPlayer": "Desolat, la vidèo a pas pogut se cargar.",
         "cantPlayInBrowser": "Aqueste navigador es pas pres en carga.",
         "cantPlayVideo": "La vidèo pòt pas èsser legida.",
-        "errorCode": "Còdi d’error",
+        "errorCode": "Còdi d'error",
```

Comment:
expected replacement of the smart quote?

Reply:
The change came from @aliciahurst 's master file; that style of quote is consistent with our other translations